.circleci/config.yml

```diff
@@ -0,0 +1,25 @@
+version: 2
+
+jobs:
+  build:
+    macos:
+      xcode: "9.0"
+
+    steps:
+      - checkout
+
+      - run:
+          name: Generate Xcode project file
+          command: swift package generate-xcodeproj
+
+      - run:
+          name: Run iOS tests
+          command: xcodebuild test -scheme SnapshotTesting-Package -destination platform="iOS Simulator,name=iPhone 8,OS=11.0"
+          environment:
+            SNAPSHOT_ARTIFACTS: $CIRCLE_ARTIFACTS
+
+      # - run:
```

Comment:
Disabled because Circle 2.0's macOS environment runs on 10.2, the WebKit APIs require 10.3, and I haven't figured out how to scope things properly with the shim.